clio-client/src/main/scala/org/broadinstitute/clio/client/webclient/ClioWebClient.scala

```diff
@@ -143,6 +146,7 @@ class ClioWebClient(
       .single(requestWithCreds)
       .via(connectionFlow)
       .initialTimeout(requestTimeout)
+      .idleTimeout(idleTimeout)
```

Comment:
If this is omitted, then there is no idle timeout, correct? If so, why is one needed now when it wasn't before?

Reply:
Same question